tests/test_data_entry_flow.py

```diff
@@ -58,7 +58,14 @@ async def async_step_init(self, user_input=None):
     assert form["errors"]["base"] == "1"
     form = await manager.async_configure(form["flow_id"])
     assert form["errors"]["base"] == "2"
-    assert len(manager.async_progress()) == 1
+    assert manager.async_progress() == [
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)